app/scripts/controllers/transactions/index.js

```diff
@@ -769,6 +772,7 @@ export default class TransactionController extends EventEmitter {
    * @returns {txMeta}
    */
```

Comment:
Parameter name `method` is not intuitive.

Reply:
Fixed here: 352f1d9054aefd84dd03b992873db4bf61cebc39